src/ground_truth_nsrts.py

```diff
@@ -282,6 +283,34 @@ def place_sampler(state: State, rng: np.random.Generator,
                       set(), option, option_vars, place_sampler)
     nsrts.add(place_nsrt)
 
+    # Place (not on any target)
+    if CFG.env == "cover_nonrefinable":
+        parameters = [block]
+        preconditions = {
+            LiftedAtom(IsBlock, [block]),
+            LiftedAtom(Holding, [block])
+        }
+        add_effects = {
```

Comment:
thank god this isn't actually a no-op, otherwise we'd have to disable the filtering in planner :P speaking of which, we might want to discuss on slack whether we should purge that whole thing now that we wiped replays